lib/react_on_rails/server_rendering_pool.rb

```diff
@@ -11,6 +11,15 @@ def self.reset_pool
       @js_context_pool = ConnectionPool.new(options) { create_js_context }
     end
 
+    def self.reset_pool_if_server_bundle_was_modified
+      return unless ReactOnRails.configuration.reload_server_js_every_request
+      file_mtime = File.mtime(ReactOnRails.configuration.server_bundle_js_file)
+      @server_bundle_timestamp ||= file_mtime
```

Comment:
so this line above says "don't reset pool if first time?"

I'd rather remove this line and have

```
return if @server_bundle_timestamp == file_mtime || @server_bundle_timestamp.nil?
```

because I read that as return if the timestamp did not change or we have not yet set the server_bundle_timestamp